dynamic-rapidpro-workspace-settings/test/env.js

```diff
@@ -0,0 +1,26 @@
+
+const { Factory } = require('rosie');
+const { faker } = require('@faker-js/faker');
```

Comment:
Nit: I don't think `rosie` should be a hard dependency since we just need to cook up a couple of field values

Reply:
Moved to _devDependencies_.